troubleshooting/tools/color-logger/README.md

```diff
@@ -0,0 +1,48 @@
+The logger will create 3 types of log events.
+
+
+1. Type 1: controlled by PAYLOAD_COUNT
+
+```
+{
+    "time": # timestamp,
+    "gauge": # a counter from 0 to PAYLOAD_COUNT - 1,
+    "uuid": "abc{gauge}",
+    "payload": "x" * PAYLOAD_SIZE
+}
+```
+
+You can search for specific events easily in CW Logs UI with
+the value of the "uuid" field. For example, search "abc77".
+
+2. Type 2: Events sent once for every 100 Type 1 events sent
+
+```
+{
+    "blue": True,
+    "id": # a counter from 0 to PAYLOAD_COUNT - 1,
+    "time": # timestamp,
+}
```

Comment:
This is for navigating a lot of logs by timestamp in lower granularity? Generalizing this, we could have a field called granularity{magnitude}: true where magnitude which is set to 100, 1000, 10000, 100000, etc which is on each log type 1. That way navigation is built into type 1 logs. To search for blue, now you would search for granularity100. (just for future efforts)

Reply:
That's a good idea